stripe/src/main/java/com/stripe/android/view/threeds2/ThreeDS2ButtonCustomization.java

```diff
@@ -0,0 +1,139 @@
+package com.stripe.android.view.threeds2;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.stripe.android.stripe3ds2.exceptions.InvalidInputException;
+import com.stripe.android.stripe3ds2.init.ui.ButtonCustomization;
+import com.stripe.android.stripe3ds2.init.ui.StripeButtonCustomization;
+import com.stripe.android.utils.ObjectUtils;
+
+/**
+ * Customization for 3DS2 buttons
+ */
+public class ThreeDS2ButtonCustomization {
```

Comment:
I'm thinking we use the prefix `Stripe3ds2` instead of `ThreeDS2`. What do you think?

Reply:
What do you think about making all of these glue classes inner classes of `PaymentAuthConfig`?

Reply:
Yup makes sense with Stripe3ds2Config in there as well